lib/arca.rb

```diff
@@ -0,0 +1,55 @@
+require_relative "arca/collector"
+require_relative "arca/model"
+require_relative "arca/report"
+require_relative "arca/callback_analysis"
+
+module Arca
+
+  # Public: Reader method for accessing the Arca::Model for analysis and
+  # reporting.
+  def self.[](klass)
+    Arca::Model.new(klass)
```

Comment:
Updated in 2b6db5da319e5ab5f1be723d386c4834a341580e and also in fcf797cb1c1db264398a502c66009129baf0af51 I now raise early if the class passed in hasn't had any data collected for it.